app/helpers/rademade_admin/form_helper.rb

```diff
@@ -92,4 +82,18 @@ def can_read_relation(data_item)
     !data_item.has_relation? || can?(:read, data_item.relation.to)
   end
 
+  def default_options
+    {
+      :wrapper => :rademade,
+      :url => @record.new_record? ? admin_create_uri(@model) : admin_update_uri(@record),
+      :as => :data,
+      :html => {
+        :multipart => true,
+        :novalidate => true,
+        :autocomplete => 'off',
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.